src/pages/EditResourcePage.tsx

```diff
@@ -28,18 +30,22 @@ interface EditResourcePageParamTypes {
 const EditResourcePage: FC = () => {
   const { t } = useTranslation();
   const { resourceIdentifierFromParam } = useParams<EditResourcePageParamTypes>();
-
+  const [resource, setResource] = useState<Resource>(emptyResource);
```

Comment:
dette er det samme som settes som `initialValues` i formik ja? er det i så fall nødvendig/ønskelig å ha den dataen to plasser? denne vil sikkert ikke endres når man endrer data i formen så det blir fort en mismatch om du ikke vil passe på selv at de er i sync, men da forsvinner litt av poenget med formik som uansett vil gjøre det

Reply:
også: `emptyResource` har sikkert type, så trenger ikke å angi igjen